src/BalanceLaws/state_types.jl

```diff
@@ -62,15 +63,22 @@ Hyper-diffusive variables
 struct Hyperdiffusive <: AbstractStateType end
 
 """
-    UpwardIntegrals <: AbstractStateType
+    UpDownIntegrals <: AbstractStateType
+
+Super-type for UpwardIntegrals and DownwardIntegrals
+"""
+abstract type UpDownIntegrals <: AbstractStateType end
```

Comment:
Why not just `IntegralType`?

Reply:
Yeah, I hadn't thought long enough. `IntegralType` sounds much better 👍